src/botPage/view/View.js

```diff
@@ -213,7 +213,9 @@ const updateTokenList = () => {
         }
 
         tokenList.forEach(tokenInfo => {
-            const prefix = isVirtual(tokenInfo) ? 'Virtual Account' : `${tokenInfo.loginInfo.currency} Account`;
+            const prefix = isVirtual(tokenInfo)
+                ? 'Virtual Account'
+                : `${tokenInfo.loginInfo.currency === 'UST' ? 'USDT' : tokenInfo.loginInfo.currency} Account`;
```

Comment:
Nit. but can we handle this better by not nesting ternary operator? It'll make the code easier to understand as well.

Reply:
changed ternary to 'if' statement